modules/libcom/src/error/errlog.h

```diff
@@ -22,8 +36,17 @@
 extern "C" {
 #endif
 
+/**
+ * errlogListener function type. 
+ * 
+ * This is used when adding or removing log listeners in ::errlogAddListener 
+ * and ::errlogRemoveListeners.
+ */
 typedef void (*errlogListener)(void *pPrivate, const char *message);
 
+/**
+ * errlog severity enums
+ */
```

Comment:
A minor nit.  Please avoid two extra lines when there is only one line of content.  eg.

```c
/** errlog severity enums */
```

Reply:
I've compressed the single line comments in the latest commit